### ISSUE TITLE
Typo in challenge "Applied Accessibility: Know When Alt Text Should be Left Blank"

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/freeCodeCamp#36225]
**Describe your problem and how to reproduce it:**

In challenge `Applied Accessibility: Know When Alt Text Should be Left Blank`, the keyword `img` should be highlighted.

**Add a Link to the page with the problem:**

https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/know-when-alt-text-should-be-left-blank

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

![Untitled](https://user-images.githubusercontent.com/25715018/59160571-91de3d80-8b01-11e9-9b31-b2296e9ab798.png)

---
File to be edited: https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/01-responsive-web-design/applied-accessibility/know-when-alt-text-should-be-left-blank.english.md
